.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDateParserConfiguration.cs

```diff
@@ -550,16 +550,33 @@ protected DateTimeResolutionResult Match2Date(Match match, DateObject referenceD
                 ret.Timex = DateTimeFormatUtil.LuisDate(year, month, day);
             }
 
-            var futureDate = DateObject.MinValue.SafeCreateFromValue(year, month, day);
-            var pastDate = DateObject.MinValue.SafeCreateFromValue(year, month, day);
-            if (noYear && futureDate < referenceDate)
+            DateObject futureDate, pastDate;
+            if (IsNonleapYearFeb29th(year, month, day))
             {
-                futureDate = futureDate.AddYears(+1);
+                futureDate = DateObject.MinValue.SafeCreateFromValue(year + 1, month, day);

```

Comment:
Yes. SafeCreateFromValue returns MinValue 0001-01-01 for invalid date. Then AddSingleDateTimeToResolution filters MinValues so that this resolution will be ignored.